custom_dc/data_cloud.md

```diff
@@ -79,30 +79,44 @@ See also the [Cloud Run](https://cloud.google.com/run/docs/create-jobs) document
 1. Click the **Container Registry** tab.
 1. Next to the **Project**, click **Change**, search for **datcom-ci** and select it.
 1. In the list of images that appears, navigate to and expand **gcr.io/datcom-ci/datacommons-data**, highlight the image you want, **stable** or **latest**, and click **Select**.
+
+   ![Cloud Run job](/assets/images/custom_dc/gcp_screenshot1.png){: width="450" }
+
 1. Optionally, in the **Job name** field, enter an alternative name as desired.
 1. In the **Region** field, select the region you chose as your location.
 1. Leave the default **Number of tasks** as 1.
 1. Expand **Container, Volumes, Connections, Security** and expand **Settings**, and set the following options:
   -  **Resources** > **Memory**: **8 GiB**
   -  **Resources** > **CPU**: **2**
-  -  **Task timeout** > **Number of retries per failed task**: **3**
-1. Click **Create** to save the job (but not run it yet).
+
+   ![Cloud Run job](/assets/images/custom_dc/gcp_screenshot2.png){: width="450" }
 
 {:.#env-vars}
 Now set environment variables:
 
 1. Click the **Variables and Secrets** tab.
 1. Click **Add variable**.
 1. Add names and values for the following environment variables:
-- `USE_CLOUDSQL`: Set to `true`.
-- `DC_API_KEY`: Set to your API key.
-- `INPUT_DIR`: Set to the Cloud Storage bucket and input folder that you created in step 2 above. 
-- `OUTPUT_DIR`: Set to the Cloud Storage bucket (and, optionally, output folder) that you created in step 2 above. If you didn't create a separate folder for output, specify the same folder as the `INPUT_DIR`.
-- `CLOUDSQL_INSTANCE`: Set to the full connection name of the instance you created in step 3 above.
-- `DB_USER`: Set to a user you configured when you created the instance in step 3, or to `root` if you didn't create a new user.
-- `DB_PASS`: Set to the user's or root password you configured when you created the instance in step 3.
-- `DB_NAME`: Only set this if you configured the database name to something other than `datacommons`.
-1. Click **Done** and **Update** to save.
+   - `USE_CLOUDSQL`: Set to `true`.
+   - `DC_API_KEY`: Set to your API key.
+   - `INPUT_DIR`: Set to the Cloud Storage bucket and input folder that you created in step 2 above. 
+   - `OUTPUT_DIR`: Set to the Cloud Storage bucket (and, optionally, output folder) that you created in step 2 above. If you didn't create a separate folder for output, specify the same folder as the `INPUT_DIR`.
+   - `CLOUDSQL_INSTANCE`: Set to the full connection name of the instance you created in step 3 above.
+   - `DB_USER`: Set to a user you configured when you created the instance in step 3, or to `root` if you didn't create a new user.
```

Comment:
Note that if they create a new user, that user will need root level permissions.

Reply:
Hmm, but what does that mean? Where do they configure that?

Reply:
Here's the answer: they get it by default https://screenshot.googleplex.com/Bg94roctC3eUTE5. So this is fine as is.